verifiers/internal/gha/provenance.go

```diff
@@ -419,6 +423,10 @@ func VerifyVersionedTag(prov iface.Provenance, expectedTag string) error {
 		return err
 	}
 
+	if ref == "" {
```

Comment:
Same as above.

Reply:
i ran into an issue where the error message jut gave empty results, because ref is empty. That happened when a user passes `--source-versioned-tag` but the build happened *not* on a release trigger.

Reply:
Ok, I think you can move this check to `utils.ValidateGitRef` and provide a better error message there.